tests/unit/test_testing.py

```diff
@@ -256,12 +258,12 @@ def test_finish_job_run_retry_no_schedule(job_store):
 
 def test_listen_for_jobs_run(job_store):
     # If we don't crash, it's enough
-    job_store.listen_for_jobs(queues=["a", "b"])
+    sync_await(job_store.listen_for_jobs(queues=["a", "b"]))
```

Comment:
I think the idea would rather be to make the test async (and mark it as such, see other async tests in this module) and to await those.

Reply:
I was hesitating. Why is it better? Anyway, I'll do it.

Reply:
3-level answer:
- Readability, I'd much rather read `await blob` than `sync_await(blob)` and then wonder where `sync_await` comes from, realizing it comes from the codebase itself, wonder if it's part of the test, etc.
- Consistency: All the other tests of async coroutines do that
- Practicality: We have a whole infrastructure around being able to launch sync functions and async coroutines, would be a shame not use it ?

Reply:
Thanks for your answer. Makes total sense.